tasks/mysql.yml

```diff
@@ -32,4 +32,4 @@
     password={{ artifactory_password }}
 
 - name: 'enable mysql driver'
-  template: src=storage_mysql.j2 dest={{artifactory_home}}/etc/storage.properties owner=root group=root mode=0644
+  template: src=storage_mysql.j2 dest={{artifactory_home}}/etc/storage.properties owner=artifactory group=artifactory mode=0600
```

Comment:
The big reason for this change was to make the file only visible by the artifactory user. So the owner and group got changed and the mode is now more restrictive. The previous mode allowed and user to see the password which is highly insecure.